pkg/crc/machine/stop.go

```diff
@@ -19,7 +19,7 @@ func (client *client) Stop() (state.State, error) {
 		return state.Error, errors.Wrap(err, "Cannot load machine")
 	}
 	if err := stopAllContainers(host, client); err != nil {
-		return state.Error, err
+		logging.Warnf("%v", err)
```

Comment:
This is not going to be a user-friendly message, since it's a warning, a nice message would be better, with the error being printed to debug.

Reply:
@cfergeau yes, I agree any specific wording you have in mind?